app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -20,6 +20,7 @@ def index
         N_("Create a filter rule. The parameters included should be based upon the filter type.")
     param :content_view_filter_id, :identifier, :desc => N_("filter identifier"), :required => true
     param :name, String, :desc => N_("package or package group: name")
+    param :names, Array, :desc => N_("packages or package groups: names")
```

Comment:
@ehelms I mostly modeled this change after the `errata_id[s]` params, so many of the design decisions were gleaned from that implementation. I don't think it is correct, however. When `errata_id` is specified, we call `rule_params.merge(filter: @filter)`, which seems correct, but when `errata_ids` is specified, we call `{errata_id: errata_id}.merge(filter: @filter)`, which disregards params such as `version`, `start_date`, etc.

1) and 2) - I hesitate to change the API; interfaces may depend on `name` being a String rather than an array of strings. Is there a way to have polymorphic params (i.e. name can be String or Array)? If so, we could add to the API without modifying the established behavior.